README.md

```diff
@@ -250,58 +262,56 @@ Connects this HyperEmitter with another one which we call a peer. Peers can exis
 will communicate over TCP using the `host` and `port` provided. An optional function can be provided that will be
 called once the connection has been made.
 
-##### _port_
-The port of the machine the peer to connect to resides on
+ * #### _port_
+ The port of the machine the peer to connect to resides on
+
+ * #### _host_
+ The host of the machine the peer to connect to resides on.
 
-##### _host_
-The host of the machine the peer to connect to resides on.
+ * #### _done_
+ A function to be called when connected.
 
-##### _done_
-A function to be called when connected.
+---
 
 <a name="listen"></a>
 ### .listen(port[, host[, done]])
 Listen on a given port/host combination. An `EventPeer` event will be
 emitter.
 
-##### _port_
-The port to listen on.
+ * #### _port_
+ The port to listen on.
 
-##### _host_
-The host to listen on.
+ * #### _host_
+ The host to listen on.
 
-##### _done_
-An optional function to be called one listening has begun.
+ * #### _done_
+ An optional function to be called one listening has begun.
 
-<a name="messages"></a>
-### .messages()
-
-The known messages, as returned by
-[protocol-buffers](http://npm.im/protocol-buffers).
+---
 
 <a name="stream"></a>
 ### .stream([opts])
 A Duplex stream to emit and receive events.
 
-##### _opts_
-An optional object of settings.
-- `from: 'beginning'` will return all the events from the beginning.
+ * #### _opts_
+ An optional object of settings.
+
+  * `from:` The point in the stream to return events since. supports 'beginning'
 
 <a name="close"></a>
 ### .close(callback)
-Close the given __hyperemitter__. After, all `emit` will return an error.
+Close a given HyperEmitter. After, all `emit` will return an error.
 
-##### _callback_
-An optional function to be called when close has completed.
+ * #### _callback_
```

Comment:
@mcollina Specifically the size? They looked a lot smaller in my markdown preview, let me make them a smaller.